i2cdrivers/src/main/java/org/openftc/i2cdrivers/QwiicLEDStrip.java

```diff
@@ -0,0 +1,190 @@
+package org.openftc.i2cdrivers;
+
+import android.graphics.Color;
+import android.support.annotation.ColorInt;
+
+import com.qualcomm.robotcore.hardware.I2cAddr;
+import com.qualcomm.robotcore.hardware.I2cDeviceSynchDevice;
+import com.qualcomm.robotcore.hardware.I2cDeviceSynchSimple;
+import com.qualcomm.robotcore.hardware.I2cWaitControl;
+import com.qualcomm.robotcore.hardware.configuration.annotations.DeviceProperties;
+import com.qualcomm.robotcore.hardware.configuration.annotations.I2cDeviceType;
+
+@I2cDeviceType()
+@DeviceProperties(name = "QWIIC LED Strip", description = "Sparkfun QWIIC LED Strip", xmlTag = "QWIIC_LED_STRIP")
+public class QwiicLEDStrip extends I2cDeviceSynchDevice<I2cDeviceSynchSimple> {
+
+    private enum Commands {
+        CHANGE_LED_LENGTH(0x70),
+        WRITE_SINGLE_LED_COLOR(0x71),
+        WRITE_ALL_LED_COLOR(0x72),
+        WRITE_RED_ARRAY(0x73),
+        WRITE_GREEN_ARRAY(0x74),
+        WRITE_BLUE_ARRAY(0x75),
+        WRITE_SINGLE_LED_BRIGHTNESS(0x76),
+        WRITE_ALL_LED_BRIGHTNESS(0x77),
+        WRITE_ALL_LED_OFF(0x78);
+        int bVal;
+
+        Commands(int bVal) {
+            this.bVal = bVal;
+        }
+    }
+
+    /**
+     * Change the color of a specific LED
+     *
+     * @param position which LED to change (1 - 255)
+     * @param color    what color to set it to
+     */
+    public void setColor(int position, @ColorInt int color) {
+        byte[] data = new byte[4];
+        data[0] = (byte) position;
```

Comment:
Assuming this is for the [Qwicc LED **Stick**](https://www.sparkfun.com/products/14783), I think the Javadoc comment is incorrect about the range for `position`, shouldn't it be 0-9?

Reply:
Looking pictures of the PCB, I see the `DO` and `CO` lines are exposed on the right side, is the idea that you would add an extra APA102 strip (like an Adafruit Dotstar) to those lines to continue the strip?

Reply:
That is exactly right.   That is why it can be up to 255.